src/kokkos/test/kokkos/HistoContainer_t.cc

```diff
@@ -0,0 +1,153 @@
+#include "KokkosCore/kokkosConfigCommon.h"
+#include "KokkosCore/kokkosConfig.h"
+#include <algorithm>
+#include <cassert>
+#include <iostream>
+#include <limits>
+#include <random>
+
+#include "KokkosCore/HistoContainer.h"
+
+template <typename T>
+void go() {
+  std::mt19937 eng;
+  std::uniform_int_distribution<T> rgen(std::numeric_limits<T>::min(), std::numeric_limits<T>::max());
+
+  constexpr int N = 12000;
+  Kokkos::View<T*,KokkosExecSpace> v_d("v_d",N);
+  typename Kokkos::View<T*,KokkosExecSpace>::HostMirror v_h("v_h",N);
+
+  constexpr uint32_t nParts = 10;
+  constexpr uint32_t partSize = N / nParts;
+//  uint32_t offsets[nParts + 1];
+
+  using Hist = HistoContainer<T, 128, N, 8 * sizeof(T), uint32_t, nParts>;
+  std::cout << "HistoContainer " << (int)(offsetof(Hist, off)) << ' ' << Hist::nbins() << ' ' << Hist::totbins() << ' '
+            << Hist::capacity() << ' ' << Hist::wsSize() << ' '
+            << (std::numeric_limits<T>::max() - std::numeric_limits<T>::min()) / Hist::nbins() << std::endl;
+
+  Kokkos::View<Hist*,KokkosExecSpace> h_d("h_d",1);
```

Comment:
I would do
```suggestion
  Kokkos::View<Hist,KokkosExecSpace> h_d("h_d");
```

Reply:
per our discussion with Kokkos team, I've change all the single objects from pointers to objects.